### PR TITLE
Issue/462/run22idr6a photozs

### PR DESCRIPTION
**Pull request by sschmidt23**
This PR adds config files for run2.2i_dr6a photo-z's, which addresses issue #462.  Parquet files have been copied to `/global/cfs/cdirs/lsst/shared/DC2-prod/Run2.2i/addons/photoz/dr6` and everything works well on tests of the composite catalog.

### CODE REVIEW

**Comment by sschmidt23** [comment]
Yes, the run2.2i_dr3 was the file that needed to be changed, I've done that in this PR to be consistent with file naming for runs 1.2, 2.1, and 2.2i_dr6.  Things should be set up correctly now for both dr6 and dr6a.

File: GCRCatalogs/catalog_configs/photoz_dc2_object_run2.2i_dr6.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+subclass_name: dc2_photoz_parquet.DC2PhotozCatalog
+base_dir: ^/DC2-prod/Run2.2i/addons/photoz/dr6
+filename_pattern: 'photoz_pdf_Run2.2i_dr6_tract_\d+\.parquet$'
+description: DC2 Run 2.2i_dr6 test BPZ Photoz table
+creators: ['Sam Schmidt, DESC DC2 team']
+addon_for: dc2_object_run2.2i_dr6
+include_in_default_catalog_list: false
```
Comment (yymao) [inline GCRCatalogs/catalog_configs/photoz_dc2_object_run2.2i_dr6.yaml:7]: `include_in_default_catalog_list` can be omitted if it's set to false.
```suggestion
```

File: GCRCatalogs/catalog_configs/dc2_object_run2.2i_dr6a_with_photoz.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    tracts: [2723, 2724, 2725, 2726, 2727, 2728, 2729, 2730, 2731, 2732, 2733, 2734, 2735,
+         2896, 2897, 2898, 2899, 2900, 2902, 2903, 2905, 2906, 2908,
+         3074, 3075, 3076, 3077, 3078, 3079, 3080, 3082, 3083, 3085, 3086,
+         3256, 3257, 3260, 3261, 3262, 3263, 3264, 3265, 3266, 3267, 3268,
+         3441, 3442, 3443, 3444, 3446, 3447, 3448, 3449, 3450, 3451, 3452, 3454,
+         3631, 3633, 3636, 3637, 3638, 3639, 3640, 3641, 3643,
+         3825, 3837,
+         4022, 4035, 4236, 4429, 4440, 4441, 4648, 4860,
+         5071, 5073]
+include_in_default_catalog_list: true
```
Comment (yymao) [inline GCRCatalogs/catalog_configs/dc2_object_run2.2i_dr6a_with_photoz.yaml:26]: For default list we only list `dr6` (`dr6a` is mainly for people who want to in the future compare `dr6b` and `dr6a`) 
```suggestion
```

**Comment by sschmidt23** [comment]
Thanks, Yao, I made the two changes.

**Pull request merged**